explorer/e2e/testFunctions.ts

```diff
@@ -1052,6 +1064,112 @@ export async function testBackpageDetails(
   return true;
 }
 
+/**
+ * Test that the Bulk Download index export workflow can be initiated and results in a download on the specified tab
+ * @param page - a Playwright page object
+ * @param tab - the tab to run the test on
+ * @returns - true if the test passes, false if the test fails but does not fail an assertion
+ */
+export async function testBulkDownloadIndexExportWorkflow(
+  page: Page,
+  tab: TabDescription
+): Promise<boolean> {
+  if (tab?.indexExportPage === undefined) {
+    console.log(
+      "testBulkIndexExportWorkflow Error: indexExportPage not specified for given tab, so test cannot run"
+    );
+    return false;
+  }
+  await page.goto(tab.url);
+  const exportButtonLocator = page.getByRole("link", {
+    name: tab.indexExportPage.indexExportButtonText,
+  });
+  await expect(exportButtonLocator).toBeVisible();
+  await exportButtonLocator.click();
+  await expect(
+    page.getByText(tab.indexExportPage.requestLandingMessage ?? "")
```

Comment:
Do we need the nullish coalescing here now that `requestLandingMessage` is a required value? If it is still necessary, I feel it might make more sense to only execute the `expect` if there is a value for `requestLandingMessage`; that way, we will never be in a situation where we're calling `page.getByText("")`. Something like:

```
const requestLandingMessage = tab.indexExportPage.requestLandingMessage;
if ( requestLandingMessage ) {
  expect(...)
}
```

Reply:
Whoops, no we don't! Thanks for catching that, I'll fix it